src/platform/sdl/sdl_window.cpp

```
@@ -6,14 +6,14 @@
 #include "canyon/graphics/sdl/sdl_graphics.h"
 
 namespace {
-    std::mutex EventFetchMutex;
-    std::list<SDL_Event> PendingEvents;
+    std::mutex EventFetchMutex;      // NOLINT(cppcoreguidelines-avoid-non-const-global-variables)
+    std::list<SDL_Event> PendingEvents; // NOLINT(cppcoreguidelines-avoid-non-const-global-variables)
 
     bool CollectSDLEventsForWindow(uint32_t windowId, std::vector<SDL_Event>* outEvents) {
         std::lock_guard lock(EventFetchMutex);
 
         SDL_Event event;
```

Clang (14.0.6)

[error] variable 'event' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)